lib/vue-lib/src/design-system/general/TruncateWithTooltip.vue

```diff
@@ -93,8 +93,12 @@ const tooltip = computed(() => {
   // we invoke forceRecompute here to force the tooltip to recompute when we need it to
   // eslint-disable-next-line @typescript-eslint/no-unused-expressions
   forceRecompute.value;
+  if (forceRecompute.value > 100) {
+    // eslint-disable-next-line vue/no-side-effects-in-computed-properties
+    forceRecompute.value = 0;
```

Comment:
Why do we need to set this back to 0? This will cause `tooltip` to be computed two times (and if we make a logic error in here, infinite loop)